<launch>  
  <!-- Launch the detection node -->
  <node pkg="aruco_gridboard" type="aruco_gridboard" name="aruco_gridboard" output="screen" >
    <param name="board_path" value="$(find aruco_gridboard)/data/layout.yaml" />
    <param name="detector_param_path" value="$(find aruco_gridboard)/data/detector_params.yml" />
    <param name="debug_display" value="false" />
   
    <remap from="/aruco_gridboard/camera_info" to="/camera/color/camera_info"/>
    <remap from="/aruco_gridboard/image_raw" to="/camera/color/image_raw"/>
  </node>

</launch>
